internal/resolution/variable_sources/entity/bundle_entity.go

```diff
@@ -154,7 +154,7 @@ func (b *BundleEntity) loadRequiredGVKs() error {
 	b.mu.Lock()
 	defer b.mu.Unlock()
 	if b.requiredGVKs == nil {
-		requiredGVKs, err := loadFromEntity[[]GVKRequired](b.Entity, property.TypeGVKRequired)
+		requiredGVKs, err := loadFromEntity[[]GVKRequired](b.Entity, property.TypeGVKRequired, false)
```

Comment:
as we discussed online, I think we should only push these booleans to constants to make it a little more readable and that's basically the cherry on top ^^

might be worth splitting the suite_test changes to a different PR. But not strictly necesssary...